### PR TITLE
Add sso project template

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
Thank you for your pull request. Please provide the following information.

---

1. **Do these changes impact *User Experience*?** (e.g., how the user interacts with Yo Office and/or the files and folders the user sees in the project that Yo Office creates)
    > 
    > * [x ]  Yes
    > * [ ]  No

    If Yes, briefly describe what is impacted.
Adding new SSO project template

2. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
    > 
    > * [a ]  Yes
    > * [ ]  No

    If Yes, briefly describe what is impacted.
Writers have been informed and made necessary documentation changes

3. **Validation/testing performed**:

    Tested on Windows and Mac

4. **Platforms tested**:

    > * [x ] Windows
    > * [x ] Mac

### CODE REVIEW

File: src/app/config/projectsJsonData.ts [hunk 1]
```diff
@@ -1,128 +1,116 @@
...
-    getHostTemplateNames()
-    {
-      let hosts : string[] = [];
-      for (let key in this.m_projectJsonData.hostTypes)
-      {
+  getHostTemplateNames(isSsoProject: boolean) {
+    let hosts: string[] = [];
+    for (let key in this.m_projectJsonData.hostTypes) {
+      if (isSsoProject) {
+        if (key === 'excel' || key === 'word' || key === 'powerpoint') {
```
Comment (akrantz) [inline src/app/config/projectsJsonData.ts]: SSO project excludes Outlook, right?

Comment (akrantz) [reply src/app/config/projectsJsonData.ts]: Might be better to say:

```
if (key == 'outlook') {
     // exclude Outlook for SSO project, so don't add host 
} else {
   host.push...
}

Comment (TCourtneyOwen) [reply src/app/config/projectsJsonData.ts]: yes that's correct for now. We tried to add it really late but were hitting some issues that reduced our confidence that we are ready to support it

Comment (akrantz) [reply src/app/config/projectsJsonData.ts]: You could combine two host.push lines so there is only one where the if is `if (isSsoProject && (key === 'outlook')) {`

Comment (TCourtneyOwen) [reply src/app/config/projectsJsonData.ts]: Just to be clear we don't support OneNote or Project either

Comment (akrantz) [reply src/app/config/projectsJsonData.ts]: It's more than just Outlook, also OneNote and Project. I think ideally the data should drive which templates are supported for which hosts. Let's put that down for a future improvement and you can leave the code for now.

**Pull request merged**